src/reynir_correct/config/TabooWords.conf

```diff
@@ -5,40 +5,249 @@
 
 # This file is included in GreynirCorrect.conf
 
-# The format is as follows, in whitespace-separated columns within each line:
-
-# 1) Taboo word + '_' + category
-# 2) Optional replacement word + '_' + category.
-#    There can be multiple replacement words,
-#    separated by tight forward slashes '/'.
-# 3) Optional explanatory comment, enclosed in double quotes.
-
-# Note that lines can be continued by ending them with a backslash '\',
-# which is especially useful for long explanatory comments.
-
-pussa_kvk				píka_kvk
-tussa_kvk				píka_kvk
-kunta_kvk				píka_kvk
-tilli_kk				limur_kk
-hommatittur_kk			hommi_kk
-faggi_kk				hommi_kk
-lessa_kvk				lesbía_kvk
-negri_kk				blökkumaður_kk
-halanegri_kk			blökkumaður_kk
-surtur_kk				blökkumaður_kk
-blámaður_kk			    blökkumaður_kk
-
-júði_kk				    gyðingur_kk         "Sá sem aðhyllist \
-                                            gyðingdóm er 'gyðingur'."
-
-múhameðstrú_kvk			íslam_hk            "Rétt heiti \
-                                            trúarbragðanna er 'íslam'."
-
-múhameðstrúarmaður_kk	múslími_kk          "Sá sem aðhyllist íslam er 'múslími'."
-
-fóstureyðing_kvk        þungunarrof_hk/meðgöngurof_hk
-
-afhomma_so                                  "Sögnin 'afhomma' getur verið meiðandi \
-                                            þar sem hún gefur í skyn að samkynhneigð sé \
-                                            eiginleiki sem unnt og/eða æskilegt \
-                                            sé að fjarlægja."
+pussa_kvk				píka_kvk					"Orðið er stundum notað yfir kynfæri kvenna og er óviðeigandi. \
```

Comment:
Undo deletion of file format instructions - they are useful and good to have in the file.

Reply:
Done!